### DISCUSSION TITLE
Best Practice to replace nested roles/include paths

### DISCUSSION THREAD

**Discussion by Hetti** [ansible/ansible-lint#3578] [Q&A] [answered]
While linting I encountered also the following warning from this issue: https://github.com/ansible/ansible-lint/issues/2750

@ssbarnea wrote in https://github.com/ansible/ansible-lint/issues/2750#issuecomment-1435074904 that those nested roles/include paths are discouraged. We used it in that way, because it made sense for us to have an easier grouping in thematic folders. 

I would like to know what is the proposed best practice? Creating really for every role a own folder and prefixing the folder with a common topic like ` server_` to achieve a somewhat grouping? would not look very practical for me but I'm open for recommendations

Thanks!

**Comment by ssbarnea** [comment, marked as answer]
Use a prefix on their names or split them in multiple collections.
